flaskapp/view_functions/ivrflow.py

```diff
@@ -79,6 +82,7 @@ def username():
             x = {"username": row.get('username')}
     return (jsonify(x))
```

Comment:
Extra parenthesis in return... see the note above

Reply:
done